_schemaV2.graphql

```diff
@@ -14546,6 +14566,18 @@ type Query {
   # A collector profile.
   collectorProfile(userID: String): CollectorProfileType
 
+  # A list of collector profiles that have sent an inquiry to a partner
+  collectorProfilesConnection(
+    after: String
+    before: String
+    first: Int
+    last: Int
+    partnerID: ID
+
+    # If present, will search by term
```

Comment:
minor/non-blocking: I wonder if the `If present, ...` here might give the idea that would also be possible to perform the query without passing a term (which is not true, in case a partnerId is passed, our endpoint in Gravity [requires it to be passed as well](https://github.com/artsy/gravity/blob/9f45bcd3dc7c4ced064aa36d4c4e3f3be8bd7e8a/app/api/v1/collector_profiles_endpoint.rb#L15))